lib/enricher/MetaData.txt

```diff
@@ -0,0 +1,767 @@
+'use strict';
```

Comment:
not sure what this is.  It is a *.txt but it looks like .js, but it has some unended block comment.